en/micro-integrator/docs/references/mediators/call-Mediator.md

```diff
@@ -47,7 +47,7 @@ When using the Call mediator in the **blocking mode** (blocking=true), enable th
 ```
 
 !!! Note
-    The call mediator in the **blocking mode** (blocking=true), builds the message from the response payload. If no response message is expected by the client, you can set the OUT_ONLY property before the call mediator to avoid building the response payload.
+    The call mediator in the **blocking mode** (blocking=true), builds the message from the response payload. If no response message is expected by the client, you can set the **OUT_ONLY** property before the call mediator to avoid building the response payload. You may set the System property **allowEmptyBodyForHttp2xx** to **true**, if you want to receive response without body for HTTP status codes 200, 201 and 202 in the blocking mode.
```

Comment:
```suggestion
    The call mediator in **blocking mode** (blocking=true) builds the message from the response payload. If a response message is not expected by the client, you can set the **OUT_ONLY** property before the call mediator to avoid building the response payload. You may set the **allowEmptyBodyForHttp2xx** system property to **true** if you want to receive a response without the body for the following HTTP status codes in blocking mode: 200, 201, and 202.
```